tests/db/test_bigchain_api.py

```diff
@@ -263,9 +263,11 @@ def test_read_transaction_invalid_block(self, b, user_vk, user_sk):
     def test_genesis_block(self, b):
         import rethinkdb as r
         from bigchaindb.util import is_genesis_block
+        from bigchaindb.db.utils import get_conn
+
         response = list(r.table('bigchain')
                         .filter(is_genesis_block)
-                        .run(b.conn))
+                        .run(get_conn()))
```

Comment:
This *was* true :smile_cat:

I'm working on #502 to move all the queries to a specific module and the `b.connection.run` might be moved out of `core.py`. In this PR I'm using the generic function to get a connection.